testsuite/forge-cli/src/main.rs

```diff
@@ -1526,13 +1518,13 @@ fn realistic_env_max_load_test(duration: Duration, test_cmd: &TestCommand) -> Fo
 
 fn realistic_network_tuned_for_throughput_test() -> ForgeConfig {
     ForgeConfig::default()
-        .with_initial_validator_count(NonZeroUsize::new(12).unwrap())
+        .with_initial_validator_count(NonZeroUsize::new(7).unwrap())
```

Comment:
I would be more comfortable with this and above change if we add weekly jobs with more nodes.  Should workflow be able to modify number of validators externally?

Reply:
> I would be more comfortable with this and above change if we add weekly jobs with more nodes.
Gotcha, this makes sense to me 😄 I'm gonna roll back these changes (to help move the PR forward), but setting up some weekly jobs with more nodes makes sense. That way, we still get signal on large swarms, but without the cost hit of running them daily.

> Should workflow be able to modify number of validators externally?
Yeah, we can probably configure this if we want. Or just add new test flavours that specify different swarm sizes? e.g., small, medium, large swarms?